gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/Oplog.java

```diff
@@ -7403,6 +7396,8 @@ else if (deferredRegions != null
             } catch(RegionDestroyedException e) {
               //This region has been destroyed, stop recovering from it.
               diskRecoveryStores.remove(diskRegionId);
+            } catch (LowMemoryException lme) {
+              diskRecoveryStores.remove(diskRegionId);
```

Comment:
lruLimitExceeded is taken care of. In case of SnappyStore any heap-based eviction is disabled from here